tests/components/deconz/test_binary_sensor.py

```diff
@@ -106,8 +96,7 @@ async def test_binary_sensors(hass):
     gateway.api.event_handler(state_changed_event)
     await hass.async_block_till_done()
 
-    presence_sensor = hass.states.get("binary_sensor.presence_sensor")
-    assert presence_sensor.state == "on"
+    assert hass.states.get("binary_sensor.presence_sensor").state == "on"
 
     await gateway.async_reset()
```

Comment:
Probably unload the integration with `hass.config_entries.async_unload`.

Reply:
Will fix in future pr

Reply:
Fixed in https://github.com/home-assistant/core/pull/41362